libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -72,6 +72,7 @@
 #define AP_ARMING_BOARD_VOLTAGE_MAX     5.8f
 #define AP_ARMING_ACCEL_ERROR_THRESHOLD 0.75f
 #define AP_ARMING_AHRS_GPS_ERROR_MAX    10      // accept up to 10m difference between AHRS and GPS
+#define AP_ARMING_MAX_DIST 500
```

Comment:
We might need a bigger default for plane.

Reply:
I think 1km for plane